app/jobs/replication/ibm_south_delivery_job.rb

```diff
@@ -2,11 +2,13 @@
 
 module Replication
   # @see Replication::IbmProvider for how S3 credentials and bucket are configured
-  # @note this class name appears in the configuration for the endpoints for which it delivers content.
+  # @note this class name appears in config files for the endpoints for which it delivers content.
   #   Please update the configs for the various environments if it's renamed or moved.
   class IbmSouthDeliveryJob < Replication::AbstractDeliveryJob
     queue_as :ibm_us_south_delivery
 
+    # perform method is defined in AbstractDeliveryJob
```

Comment:
I'm 50/50 on whether this comment is more helpful than it is noisy. Could go either way.

Reply:
agree that this comment isnt necessary, but personally I found it weird that there was no perform method in these classes and had to go look in the superclass.   I could also go either way.

Reply:
K, let's leave them in then